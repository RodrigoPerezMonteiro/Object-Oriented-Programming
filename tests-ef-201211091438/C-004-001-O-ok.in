testC2.dat
2
vader@imperial.net
salada mista
4
2
kirk@starfleet.net
linguado grelhado
2
3
kirk@starfleet.net
2
kirk@starfleet.net
salada mista
3
3
kirk@starfleet.net
2
kirk@starfleet.net
alface
9
0
testC4b.dat
