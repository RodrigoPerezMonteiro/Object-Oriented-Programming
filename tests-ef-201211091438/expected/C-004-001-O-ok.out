Ficheiro a abrir: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Prato do dia a encomendar (nome): Número de doses a encomendar: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Prato do dia a encomendar (nome): Número de doses a encomendar: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Prato do dia a encomendar (nome): Número de doses a encomendar: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Prato do dia a encomendar (nome): Número de doses a encomendar: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Ficheiro a guardar: 